tests/python/tests/test_trino.py

```
@@ -774,6 +774,30 @@ def test_table_extra_properties(trino, warehouse: conftest.Warehouse):
+    cur.execute(
+        f"CREATE TABLE {ns}.my_table (my_ints INT, my_floats DOUBLE, strings VARCHAR) WITH (format='PARQUET')"
+    )
+    cur.execute(f"INSERT INTO {ns}.my_table VALUES (1, 1.0, 'a'), (2, 2.0, 'b')")
+    cur.execute(
+        f"CREATE OR REPLACE VIEW {ns}.inner_view AS SELECT strings FROM {ns}.my_table"
+    )
+    cur.execute(
+        f"CREATE OR REPLACE VIEW {ns}.outer_view AS SELECT strings FROM {ns}.inner_view"
+    )
+
+    r = cur.execute(f"SELECT * FROM {ns}.outer_view").fetchall()
```

Ruff (0.15.9)

[error] Possible SQL injection vector through string-based query construction

(S608)